Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.h

```diff
@@ -79,7 +87,9 @@ class ControlGroup
   const std::string name;
   const std::string ui_name;
   const GroupType type;
+  const bool can_be_disabled;
 
+  bool enabled;
```

Comment:
I'd set enabled to `true` here, then remove them from the constructors.  This ensure any future contructors will have `true` as the default.